### PR TITLE
Update supported distributions

### PR DESCRIPTION
**Pull request by aahlenst**
* Ubuntu: Drop Disco, add Groovy
* Fedora: Drop 30, add 33

### CODE REVIEW

File: linux/README.md [hunk 1]
```diff
@@ -181,16 +181,16 @@ By specifying all build properties (see above) building and uploading can be don
...
-| JDK 12 (Hotspot, OpenJ9) | 8 (jessie), 9 (stretch), 10 (buster) | 16.04 (xenial), 18.04 (bionic), 19.04 (disco), 19.10 (eoan), 20.04 (focal) |
-| JDK 13 (Hotspot, OpenJ9) | 8 (jessie), 9 (stretch), 10 (buster) | 16.04 (xenial), 18.04 (bionic), 19.04 (disco), 19.10 (eoan), 20.04 (focal) |
-| JDK 14 (Hotspot, OpenJ9) | 8 (jessie), 9 (stretch), 10 (buster) | 16.04 (xenial), 18.04 (bionic), 19.04 (disco), 19.10 (eoan), 20.04 (focal) |
-| JDK 15 (Hotspot, OpenJ9) | 8 (jessie), 9 (stretch), 10 (buster) | 16.04 (xenial), 18.04 (bionic), 19.04 (disco), 19.10 (eoan), 20.04 (focal) |
+| OpenJDK                  | Debian                               | Ubuntu                                                                      |
+|--------------------------|--------------------------------------|-----------------------------------------------------------------------------|
+| JDK 8 (Hotspot, OpenJ9)  | 8 (jessie), 9 (stretch), 10 (buster) | 16.04 (xenial), 18.04 (bionic), 19.10 (eoan), 20.04 (focal), 20.10 (groovy) |
+| JDK 9 (Hotspot, OpenJ9)  | 8 (jessie), 9 (stretch), 10 (buster) | 16.04 (xenial), 18.04 (bionic), 19.10 (eoan), 20.04 (focal), 20.10 (groovy) |
```
Comment (karianna) [inline linux/README.md:187]: We don't support 19.04?

File: linux/README.md [hunk 2]
```diff
@@ -201,14 +201,14 @@ All packages can be installed on Amazon Linux, CentOS, Fedora, Red Hat Enterpris
...
-| JDK 8 (Hotspot, OpenJ9)  | 1, 2    | 6, 7, 8 | 30, 31, 32 | 6, 7, 8 | 15.1, 15.2 | 12, 15 |
-| JDK 9 (Hotspot, OpenJ9)  | 1, 2    | 6, 7, 8 | 30, 31, 32 | 6, 7, 8 | 15.1, 15.2 | 12, 15 |
-| JDK 10 (Hotspot, OpenJ9) | 1, 2    | 6, 7, 8 | 30, 31, 32 | 6, 7, 8 | 15.1, 15.2 | 12, 15 |
-| JDK 11 (Hotspot, OpenJ9) | 1, 2    | 6, 7, 8 | 30, 31, 32 | 6, 7, 8 | 15.1, 15.2 | 12, 15 |
-| JDK 12 (Hotspot, OpenJ9) | 1, 2    | 6, 7, 8 | 30, 31, 32 | 6, 7, 8 | 15.1, 15.2 | 12, 15 |
-| JDK 13 (Hotspot, OpenJ9) | 1, 2    | 6, 7, 8 | 30, 31, 32 | 6, 7, 8 | 15.1, 15.2 | 12, 15 |
-| JDK 14 (Hotspot, OpenJ9) | 1, 2    | 6, 7, 8 | 30, 31, 32 | 6, 7, 8 | 15.1, 15.2 | 12, 15 |
-| JDK 15 (Hotspot, OpenJ9) | 1, 2    | 6, 7, 8 | 30, 31, 32 | 6, 7, 8 | 15.1, 15.2 | 12, 15 |
+| JDK 8 (Hotspot, OpenJ9)  | 1, 2    | 6, 7, 8 | 31, 32, 33 | 6, 7, 8 | 15.1, 15.2 | 12, 15 |
+| JDK 9 (Hotspot, OpenJ9)  | 1, 2    | 6, 7, 8 | 31, 32, 33 | 6, 7, 8 | 15.1, 15.2 | 12, 15 |
```
Comment (karianna) [inline linux/README.md:205]: No longer supporting 30?

File: linux/README.md (hunk 1 above)
Comment (aahlenst) [reply linux/README.md:187]: Has reached its EOL in January, see https://wiki.ubuntu.com/Releases.

File: linux/README.md (hunk 2 above)
Comment (aahlenst) [reply linux/README.md:205]: Has reached its EOL a few months ago, too. The Fedora Wiki does not specify an exact date, but it's no longer on the list of supported distributions. See https://fedoraproject.org/wiki/Releases.

**Pull request merged**